server/services/flightGenerator.js

```diff
@@ -0,0 +1,217 @@
+/** @format */
+
+// random generators
+const randomPrice = (min, max) => {
+	return Math.floor(Math.random() * (max - min) + min);
+};
+const randomInt = (num) => {
+	return Math.floor(Math.random() * num);
+};
+
+// times for direct flights
+const noStops = [
+	{
+		depart: "6:50AM",
+		arrive: "11:10AM",
+		duration: "4 hr 20 min",
+	},
+	{
+		depart: "11:25AM",
+		arrive: "4:55PM",
+		duration: "5 hr 30 min",
+	},
+	{
+		depart: "2:30PM",
+		arrive: "9:15PM",
+		duration: "6 hr 45 min",
+	},
+	{
+		depart: "4:15PM",
+		arrive: "8:25PM",
+		duration: "4 hr 10 min",
+	},
+];
+const directTime = () => {
+	const direct = noStops[randomInt(noStops.length)];
+	return {
+		DepartureTime: direct.depart,
+		ArrivalTime: direct.arrive,
+		Duration: direct.duration,
+	};
+};
+// times for indirect flights
+const withStops = [
+	{
+		depart: "6:50AM",
+		arrive: "11:10AM",
+		duration: "4 hr 20 min",
+		stopDuration: "1 hr 45 min",
+	},
+	{
+		depart: "11:25AM",
+		arrive: "4:55PM",
+		duration: "5 hr 30 min",
+		stopDuration: "3 hr 0 min",
+	},
+	{
+		depart: "2:30PM",
+		arrive: "9:15PM",
+		duration: "6 hr 45 min",
+		stopDuration: "4 hr 10 min",
+	},
+	{
+		depart: "4:15PM",
+		arrive: "8:25PM",
+		duration: "4 hr 10 min",
+		stopDuration: "1 hr 40 min",
+	},
+];
```

Comment:
this feature is already good as is. Just noting that in the future if you wanted to get even more interesting with the generator, we can have randomly generated times (ie. not picking from a list) with dynamic durations, etc.